src/mmfatl.c

```diff
@@ -92,7 +92,7 @@ struct FatalErrorBufferDescriptor const* getFatalErrorBufferDescriptor()
  *   requires also information about available memory.
  * \param aDescriptor [null] a pointer to a \ref FatalErrorBufferDescriptor
  *   under investigation.
- * \returns 1 if acceptable, 0 else.
+ * \returns 1 if acceptabe, 0 else.
```

Comment:
typo

Reply:
Still someone watching...  Thanks

Reply:
Unfortunately, I cannot (yet) verify the correctness of the code...